ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerFactory.scala

```diff
@@ -95,24 +95,27 @@ object ParallelIndexerFactory {
       implicit val ec: ExecutionContext = resourceContext.executionContext
       haCoordinator.protectedExecution(connectionInitializer =>
         initializeHandle(
-          DbDispatcher
-            .owner(
-              // this is the DataSource which will be wrapped by HikariCP, and which will drive the ingestion
-              // therefore this needs to be configured with the connection-init-hook, what we get from HaCoordinator
-              dataSource = dataSourceStorageBackend.createDataSource(
-                jdbcUrl = jdbcUrl,
-                dataSourceConfig = dataSourceConfig,
-                connectionInitHook = Some(connectionInitializer.initialize),
-              ),
-              serverRole = ServerRole.Indexer,
-              connectionPoolSize =
-                ingestionParallelism + 1, // + 1 for the tailing ledger_end updates
-              connectionTimeout = FiniteDuration(
-                250,
-                "millis",
-              ), // 250 millis is the lowest possible value for this Hikari configuration (see HikariConfig JavaDoc)
-              metrics = metrics,
-            )
+          for {
+            dbDispatcher <- DbDispatcher
+              .owner(
+                // this is the DataSource which will be wrapped by HikariCP, and which will drive the ingestion
+                // therefore this needs to be configured with the connection-init-hook, what we get from HaCoordinator
+                dataSource = dataSourceStorageBackend.createDataSource(
+                  jdbcUrl = jdbcUrl,
+                  dataSourceConfig = dataSourceConfig,
+                  connectionInitHook = Some(connectionInitializer.initialize),
+                ),
+                serverRole = ServerRole.Indexer,
+                connectionPoolSize =
+                  ingestionParallelism + 1, // + 1 for the tailing ledger_end updates
+                connectionTimeout = FiniteDuration(
+                  250,
+                  "millis",
+                ), // 250 millis is the lowest possible value for this Hikari configuration (see HikariConfig JavaDoc)
```

Comment:
May be: 
```
import scala.concurrent.duration._
250.millis
```

Reply:
> May be:
> import scala.concurrent.duration._
@skisel-da - the above code was only indented as part of this PR.  I don't like a lot of the choices the auto code layout make, in the example above if I was editing the code I may move the comment to be above the line.  That said I don't want to get into fixing layout problems as part of this PR.

Reply:
Thanks for the explanation @simonmaxen-da